change/@microsoft-teams-js-6a9e9685-b682-40f3-abe9-3a5c94c35f21.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Added `location`, `geoLocation` and `permissions` to `teamsRuntimeConfig` to return true for isSupported in geolocation in v1",
```

I would probably rephrase this so that it makes more sense to external developers. Maybe something like 
```suggestion
  "comment": "Added support for `location`, `geoLocation` and `permissions` to Teams v1",
```